setup.cake

```diff
@@ -0,0 +1,106 @@
+        if(name.EndsWith("Cake.Bakery") || name.EndsWith("Tests") || name.EndsWith("Cake.Scripting"))
+        {
+            continue;
+        }
+
+        DotNetCorePack(project.FullPath, new DotNetCorePackSettings {
+            NoBuild = true,
+            Configuration = BuildParameters.Configuration,
+            OutputDirectory = BuildParameters.Paths.Directories.NuGetPackages,
+            ArgumentCustomization = args => args
+                .Append("/p:Version={0}", BuildParameters.Version.SemVersion)
+                .Append("/p:AssemblyVersion={0}", BuildParameters.Version.Version)
+                .Append("/p:FileVersion={0}", BuildParameters.Version.Version)
+                .Append("/p:AssemblyInformationalVersion={0}", BuildParameters.Version.InformationalVersion)
+        });
+    }
+
+    var net46FullArtifactPath = MakeAbsolute(net46ArtifactPath).FullPath;
+    var net46FullArtifactPathLength = net46FullArtifactPath.Length+1;
+
+    // Cake.Bakery - .NET 4.6
+    NuGetPack("./nuspec/Cake.Bakery.nuspec", new NuGetPackSettings {
+        Version = BuildParameters.Version.SemVersion,
+        //ReleaseNotes = TODO,
+        BasePath = net46ArtifactPath,
+        OutputDirectory = BuildParameters.Paths.Directories.NuGetPackages,
+        Symbols = false,
+        NoPackageAnalysis = true,
+        Files = GetFiles(net46FullArtifactPath + "/**/*")
+                                .Select(file=>file.FullPath.Substring(net46FullArtifactPathLength))
+                                .Select(file=> !file.Equals("LICENSE") ? 
+                                    new NuSpecContent { Source = file, Target = "tools/" + file } :
+                                    new NuSpecContent { Source = file, Target = file })
+                                .ToArray()
+    });
+
+    var netCoreFullArtifactPath = MakeAbsolute(netCoreArtifactPath).FullPath;
+    var netCoreFullArtifactPathLength = netCoreFullArtifactPath.Length+1;
+
+    // Cake.Bakery - .NET Core
+    NuGetPack("./nuspec/Cake.Bakery.CoreCLR.nuspec", new NuGetPackSettings {
+        Version = BuildParameters.Version.SemVersion,
+        //ReleaseNotes = TODO,
+        BasePath = netCoreArtifactPath,
+        OutputDirectory = BuildParameters.Paths.Directories.NuGetPackages,
+        Symbols = false,
+        NoPackageAnalysis = true,
+        Files = GetFiles(netCoreFullArtifactPath + "/**/*")
+                                .Select(file=>file.FullPath.Substring(netCoreFullArtifactPathLength))
+                                .Select(file=> !file.Equals("LICENSE") ? 
+                                    new NuSpecContent { Source = file, Target = "tools/" + file } :
+                                    new NuSpecContent { Source = file, Target = file })
+                                .ToArray()
+    });
+});
+
+// Disable default Pack task and exchange with ours
+BuildParameters.Tasks.DotNetCorePackTask.WithCriteria(() => false);

```

Comment:
@mholo65 see the example that I linked to in Slack.  This can be done slightly differently.

https://github.com/cake-contrib/Cake.Codecov/blob/using-cake-recipe/setup.cake#L30